test/musica/tuvx/test_tuvx_surface_albedo.F90

```diff
@@ -0,0 +1,57 @@
+program test_tuvx_surface_albedo
+
+  use musica_ccpp_tuvx_surface_albedo
+
+  implicit none
+
+#define ASSERT(x) if (.not.(x)) then; write(*,*) "Assertion failed[", __FILE__, ":", __LINE__, "]: x"; stop 1; endif
+#define ASSERT_NEAR( a, b, abs_error ) if( (abs(a - b) >= abs_error) .and. (abs(a - b) /= 0.0) ) then; write(*,*) "Assertion failed[", __FILE__, ":", __LINE__, "]: a, b"; stop 1; endif
+
+  call test_update_surface_albedo()
+
+contains
+
+  subroutine test_update_surface_albedo()
+    use musica_ccpp_tuvx_wavelength_grid, only: create_wavelength_grid
+    use musica_util,                      only: error_t
+    use musica_tuvx_grid,                 only: grid_t
+    use musica_tuvx_profile,              only: profile_t
+    use ccpp_kinds,                       only: kind_phys
+
+    integer, parameter       :: NUM_WAVELENGTH_BINS = 4
+    real, parameter          :: ABS_ERROR = 1e-4
+    real(kind_phys)          :: wavelength_grid_interfaces(NUM_WAVELENGTH_BINS + 1) = &
+        [200.0e-9_kind_phys, 210.0e-9_kind_phys, 240.0e-9_kind_phys, 300.0e-9_kind_phys, 400.0e-9_kind_phys]
+    real(kind_phys)          :: host_surface_albedo = 500.5_kind_phys
```

Comment:
Is this a reasonable value? Albedo values are generally in the range of [0, 1]. I'm not sure if this will be passed to TUV-x for testing in the future so it may be good to have more realistic values passed in.

Reply:
I agree, thank you for the suggestion!